### PR TITLE
ConfParserTest.py: Fix typos in comments

### PR DESCRIPTION
**Pull request by haxzie**
This fixes multiple occurences of typos in the comments
omment --> comment
inexistent --> non-existent

Fixes https://github.com/coala/coala/issues/5785

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: tests/parsing/ConfParserTest.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def test_parse_nonexisting_file(self):
     def test_parse_nonexisting_section(self):
         self.assertRaises(IndexError,
                           self.uut.get_section,
-                          'inexistent section')
+                          'nonexistent section')
```
Comment (ayan-b) [inline tests/parsing/ConfParserTest.py]: A hyphen between non and existent, _i.e._ non-existent? [Ref](https://en.oxforddictionaries.com/definition/non-existent)

**Comment by ayan-b** [comment]
For typo fixes generally `-->` is used instead of `->` ([Ref](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html)). Please change your commit message.

**Comment by haxzie** [comment]
@ayan-b Fixed, thanks for the pointers :smile:

**Comment by ayan-b** [comment]
> @ayan-b Fixed, thanks for the pointers smile

Looks good, just edit the PR message as well, _i.e._ here: https://github.com/coala/coala/pull/5789#issue-220962854 .

**Comment by haxzie** [comment]
@ayan-b Sure, consider it done :+1:

**Comment by pandyamarut** [comment]
@haxzie  I Doubt. 
Bot has an issue.

**Comment by jayvdb** [comment]
Also please manually rebase your commit.

**Comment by haxzie** [comment]
@jayvdb sorry about the commit message. I referred the coala's commit guidelines and found this example.

```
configure.py: Fix docstring typo 

This fixes the typo and changes it from wether --> whether. 

Fixes https://github.com/coala/coala/issues/4018
```

So, should it be `closes` ?

**Comment by akshatkarani** [comment]
> @jayvdb sorry about the commit message. I referred the coala's commit guidelines and found this example.
> 
> ```
> configure.py: Fix docstring typo 
> 
> This fixes the typo and changes it from wether --> whether. 
> 
> Fixes https://github.com/coala/coala/issues/4018
> ```
> So, should it be `closes` ?

Yes

**Comment by haxzie** [comment]
@akshatkarani done :+1:

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5789/commits/87a809336adcb79a0b5b370ccdf0ba9b34213ec8

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**